### ISSUE TITLE
Theme not showing up after git clone?

### ISSUE THREAD

**Issue by jwvanderbeck** [kelyvin/caffeine-theme#48]
I did a git clone to get the theme and I see it in my themes directory alongside the default Casper.  But when I go to settings, only Casper shows up in the list.

**Comment by kelyvin** [comment]
Hi @jwvanderbeck, 

Off the top of my head I'm not sure what the issue may be or how to debug this without more context. So as a quick text, can you try downloading the latest release (https://github.com/kelyvin/caffeine-theme/releases/tag/v2.9.0), unzip the folder within your `themes` directory, and see if that changes anything?

**Comment by jwvanderbeck** [comment]
Ok so since I was downloading it anyway I figured I would just use the "Upload Theme" option in the settings and gave it the zip file I had downloaded.  After processing it said this:

> "caffeine-theme-v2.9.0" uploaded successfully but some warnings were generated...
package.json file should be present."

And then it activated.  after activating the new theme, then suddenly the one I had got with git clone showed up as well lol.

Maybe that missing package.json is the issue?
